Tests/js/views/services/plugins/assets/ez-locationsearchplugin-tests.js

```diff
@@ -274,12 +321,13 @@ YUI.add('ez-locationsearchplugin-tests', function (Y) {
             delete this.plugin;
             delete this.capi;
             delete this.contentService;
-            delete this.query;
+            delete this.contentQuery;
+            delete this.locationQuery;
         },
 
         "Should load Content and ContentType into location struct": function () {
             var contentInfo = this.contentInfo,
-                response = this.response,
+                response = this.locationResponse,
                 resultAttr = 'whateverAttr';
 
             Y.eZ.Content = Y.Model;
```

Comment:
since you are now injecting a custom Content constructor, I guess this line can/should be remove.